### ISSUE TITLE
CDN outdated

### ISSUE THREAD

**Issue by ff6347** [dariaDunkelbier/Projekt-1-2015-11-09#1]
Hey @dariaDunkelbier 

Great you start working on it. I think your CDN uses an outdated version of P5.js. Take a look here this should be always the latest stable version. 

http://cdnjs.com/libraries/p5.js

**Comment by dariaDunkelbier** [comment]
Yes that was such a hustle.. All the People in the Hostel did ask me, what the hell i am doing there and i answered: i really dont know:)
Thank you for the Link i will check it later! 
Code #2 is that a name suggestion?
Grüße aus New york 

Von meinem iPhone gesendet

**Comment by ff6347** [comment]
Ask me on the chat or in the workspace if you need some help.
